facility-all-programs.ipynb

```diff
@@ -731,7 +791,7 @@
     "## This section saves the facility data to your computer.\n",
     "The first of the next two cells ask you to select one of the data sets to export from the dropdown list.  The next cell actually exports that data to your computer.  You can return to the dropdown to export additional data files.\n",
     "\n",
-    "Note: When you click on [] in the second cell, it may continue to show \\*. That's to be expected! Check your Downloads folder and confirm that the spreadsheet was succesfully exported. Hit the square button (Interrupt Kernel) at the top of the page. You can now choose to export other data sets from the dropdown..."
+    "_Note: When you click on [] in the second cell, it may continue to show \\*. That's to be expected! Check your Downloads folder and confirm that the spreadsheet was succesfully exported. Hit the square button (Interrupt Kernel) at the top of the page. You can now choose to export other data sets from the dropdown..._"
```

Comment:
In general I've found that `*` renders more consistently across platforms than `_`